kubernetes/README.md

```diff
@@ -116,3 +116,5 @@ $ VM_NAME=xXx \
   && ansible-playbook ansible/playbook/sec_host.yml -e vm_name=${VM_NAME} -e provider=hetzner \
   && ansible-playbook kubernetes/ansible/k8s.yml --limit ${VM_NAME}
 ```
+
+> NOTE: Both kubernetes playbooks (`k8s` and `k8s-misc`) can have it's host overriden using the `override_host` variable, e.g., `-e override_host=localhost` to launch it on the controller itself.
```

Comment:
Can we use `host` instead of `controller` as a controller means for a lot of us an application doing a specific job ?

Reply:
I'm trying to use Ansible naming convention where they name the machine that launches ansible as the *controller* or *control node* and the hosts against which the actions are taken as *host* or *managed nodes*. 

@cmoulliard  would it work better if I changed from *controller* it to *control node*?

https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html

Reply:
> _control node_?

No :-( So keep it then `controller` name